fili-core/src/main/java/com/yahoo/bard/webservice/metadata/SegmentInfo.java

```diff
@@ -59,6 +66,36 @@ public Interval getInterval() {
     }
 
     /**
+     * Getter for the dimensions in this segment.
+     *
+     * @return The list of dimension names
+     */
+    public List<String> getDimensions() {
+        return dimensions;
+    }
+
+    /**
+     * Getter for the metrics in this segment.
+     *
+     * @return The list of metric names
+     */
+    public List<String> getMetrics() {
+        return metrics;
+    }
+
+    /**
+     * Getter for list of dimension and metric names.
+     *
+     * @return The list of dimension and metric names
+     */
+    public List<String> getColumnNames() {
```

Comment:
It is all getter methods, I guess that is why we don't already have a test for it.